compliance-web/src/components/App/Inspections/InspectionDrawer.tsx

```diff
@@ -64,13 +64,12 @@ const InspectionDrawer: React.FC<InspectionDrawerProps> = ({
   const { data: attendanceList } = useAttendanceOptionsData();
   const { data: agenciesList } = useAgenciesData();
   const { data: firstNationsList } = useFirstNationsData();
-  const { data: inattendanceOfficersList } = useStaffUsersData(true)
+  const { data: inattendanceOfficersList } = useStaffUsersData(true);
   const currentUser = useCurrentLoggedInUser();
 
-  const staffUserList = [
-    caseFile.primary_officer,
-    ...(caseFile.officers ?? []),
-  ].filter(Boolean) as StaffUser[];
+  const staffUserList = [caseFile.primary_officer, ...(caseFile.officers ?? [])]
+    .filter(Boolean)
+    .sort((a, b) => a?.name.localeCompare(b?.name ?? "") ?? 0) as StaffUser[];
```

Comment:
Would it be better to have this logic in the Filter component and pass the attribute name by which the list has to be filtered?

Reply:
this is only in that sorting of Autocomplete, not the table filter, the table filter sorting is handled in the corresponding component